### PR TITLE
Add governance script to wrap Morpho tokens

### PR DESCRIPTION
**Pull request by shahthepro**
## Governance Proposal
- Proposal ID: [52303668283507532886105041405617076369748861896782994594114630508874108983718](https://originprotocol.eth.limo/#/more/1:0x1d3fbd4d129ddd2372ea85c5fa00b2682081c9ec:52303668283507532886105041405617076369748861896782994594114630508874108983718)
- Propose Tx: [0xdff1b4aa688bebb17c94b69c1f28e33718c5a48b284966e0b6f3e7c9c767bfe0](https://etherscan.io/tx/0xdff1b4aa688bebb17c94b69c1f28e33718c5a48b284966e0b6f3e7c9c767bfe0)
## Governance actions:
- Approve wrapper contract to move Morpho
- Wrap all legacy Morpho tokens on the Timelock and send them to the 2/8 multisig

## Deploy checklist

Two reviewers complete the following checklist:

```
- [ ] Governance proposal matches the deploy script
- [ ] Smoke tests pass after fork test execution of the governance proposal
```

### CODE REVIEW

**Review by sparrowDom — DISMISSED** [review]
I've verified that: 
- [x] Governance proposal matches the deploy script
- [x] Morpho legacy token contract & Morpho depositor contract addresses are correct

File: contracts/deploy/mainnet/111_morpho_wrap_and_transfer.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    deployerIsProposer: false,
+    proposalId:
+      "52303668283507532886105041405617076369748861896782994594114630508874108983718",
+  },
+  async () => {
+    const { strategistAddr } = await getNamedAccounts();
+
+    const cLegacyMorpho = await ethers.getContractAt(
+      [
+        "function approve(address,uint256) external",
```
Comment (naddison36) [inline contracts/deploy/mainnet/111_morpho_wrap_and_transfer.js:19]: this is a nice way of loading an external contract that we will probably never use again

File: contracts/deploy/mainnet/111_morpho_wrap_and_transfer.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        "function balanceOf(address) external view returns(uint256)",
+      ],
+      "0x9994E35Db50125E0DF82e4c2dde62496CE330999"
+    );
+
+    const cWrapperContract = await ethers.getContractAt(
+      ["function depositFor(address,uint256) external"],
+      "0x9D03bb2092270648d7480049d0E58d2FcF0E5123"
```
Comment (naddison36) [inline contracts/deploy/mainnet/111_morpho_wrap_and_transfer.js:27]: I doubled checked this as TG had 0x17C5b2eB6C72F86C98b8f3ACEc3ea47947aeC45a for the Morpho Wrapper contract. But Morpho's Notion doc had 0x9D03bb2092270648d7480049d0E58d2FcF0E5123 as per the deploy script 
https://morpho-labs.notion.site/New-MORPHO-token-136d69939e6d8047a73cd32a244f5ed7

Both are funded with new Morpho tokens but only 0x9D03bb2092270648d7480049d0E58d2FcF0E5123 is being used.

So the deploy script it good. No changes required

**Review by naddison36 — APPROVED** [review]
Looks good

**Comment by naddison36** [comment]
I've verified the deployment
- [x] Governance proposal matches the deploy script
- [x] Smoke tests pass after fork test execution of the governance proposal

**Pull request merged**